CHECLabPy/spectrum_fitters/gentile.py

```diff
@@ -15,13 +15,18 @@ def __init__(self, n_illuminations, config_path=None):
         ----------
         n_illuminations : int
             Number of illuminations to fit simultaneously
+        fit_range : [float,float]
+            A tuple or list with [low,high] values of the range on which the fit 
+            should be done on   
         """
         super().__init__(n_illuminations, config_path)
 
         self.nbins = 100
         self.range = [-40, 150]
+        if(fit_range is not None):
+            self.range = fit_range
 
-        self.add_parameter("norm", None, 0, 100000, fix=True, multi=True)
+        self.add_parameter("norm", None, 0, 1e5, fix=False, multi=True)
```

Comment:
Could you restore this line?
Parameters can also be changed via the config file. To achieve these changes you did here I would include the following in the config file:
```
limits:
  norm0:
  - 0
  - 1e5
fix:
  norm0: false
```

Reply:
I'll do forgot to put them back before committing.